src/services/fetchCarouselItemsFunction.js

```diff
@@ -0,0 +1,11 @@
+const API_BASE_URL = import.meta.env.VITE_BACKEND_URL;
+
+async function fetchCarouselItemsFunction (setCarouselItems, carouselCategory) {
+	await fetch(`${API_BASE_URL}/${carouselCategory}`)
+		.then(res => res.json())
```

Comment:
what if request fails with some HTTP Status Code? 
in this scenario we won't be able to do `res.json()`